Algorithm.Framework/Alphas/AlphaModelPythonWrapper.cs

```diff
@@ -41,7 +41,7 @@ public string Name
                     }
 
                     // if the model does not define a name property, use the python type name
-                    return _model.GetPythonType();
+                    return _model.__class__.__name__;
```

Comment:
Awesome! Thank you for fixing, I wasn't convinced it was correct before.

Reply:
Me neither! We could have `_model.GetPythonType().ToString()` instead, but it returns something like `<class 'RsiAlphaModel.RsiAlphaModel'>` and we would have to manipulate this string. Then I look for an alternative and that solution pop out.